plasmapy/transport/classical/epperlein_haines.py

```diff
@@ -0,0 +1,238 @@
+        i = _find_nearest(self.Z)
+        return β0[i]*np.ones(self.chi.size)
+    
+    def norm_beta_perp(self):
+        i = _find_nearest(self.Z)
+        return (β1p[i]*self.chi + β0p[i]) / (self.chi**3 + b2p[i]*self.chi**2 + b1p[i]*self.chi + b0p[i] )**(8/9)
+    
+    # TODO: Note that this function doesn't match the EH paper Fig. 1 in the
+    # chi -> inf side. The coefficients and polynomial are right...
+    # this might be a mistake in the EH tables?
+    def norm_beta_cross(self):
+        i = _find_nearest(self.Z)
+        return self.chi*(β1pp[i]*self.chi + β0pp[i]) /  \
+                    (self.chi**3 * b2pp[i]*self.chi**2 + b1pp[i]*self.chi + b0pp[i])
+                    
+    def norm_kappa_para(self):
+        i = _find_nearest(self.Z)
+        return γ0[i]*np.ones(self.chi.size)
+                    
+    def norm_kappa_perp(self):
+        i = _find_nearest(self.Z)
+        return (γ1p[i]*self.chi + γ0p[i])/ \
+                    (self.chi**3 + c2p[i]*self.chi**2 + c1p[i]*self.chi + c0p[i])
+    
+    def norm_kappa_cross(self):
+        i = _find_nearest(self.Z)
+        return self.chi*(γ1pp[i]*self.chi + γ0pp[i])/  \
+                 (self.chi**3 + c2pp[i]*self.chi**2 + c1pp[i]*self.chi + c0pp[i])
+
+
+
+
+
+
+if __name__ == '__main__':
+    chi = np.linspace(-2, 2, num=100)
+    chi = 10**chi
+
+    # Instantiate the object
+    coef1 = EpperleinHainesCoefficents(chi, 1)
+    coef2 = EpperleinHainesCoefficents(chi, np.inf)
+
+    fig, axarr = plt.subplots(nrows=3, ncols=2, figsize=(10,10), sharex=True)
+    
+    for i in range (3):
+        for j in range(2):
+            ax = axarr[i][j]
+            ax.set_xscale("log")
+            ax.set_yscale("log")
+            ax.set_xlim(1e-2, 1e2)
+        
+    axarr[-1][0].set_xlabel("$\\chi$")
+    axarr[-1][1].set_xlabel("$\\chi$")
+    
+    ax = axarr[0][0]
+    ax.set_title('$\\alpha_\\perp$')
+    ax.set_ylim(0.2, 1)
+    ax.plot(chi, coef1.norm_alpha_perp(), label="Z=1")
+    ax.plot(chi, coef2.norm_alpha_perp(),  label="Z=Infinite")
+    ax.legend()
+    
+    ax = axarr[1][0]
+    ax.set_title('$\\alpha_\\wedge$')
+    ax.set_ylim(0.01, 1)
+    ax.plot(chi, coef1.norm_alpha_cross())
+    ax.plot(chi, coef2.norm_alpha_cross())
+    
+    ax = axarr[2][0]
+    ax.set_title('$\\beta_\\perp$')
+    ax.set_ylim(0.001, 10)
+    ax.plot(chi, coef1.norm_beta_perp())
+    ax.plot(chi, coef2.norm_beta_perp())
+    
+    
+    ax = axarr[0][1]
+    ax.set_title('$\\beta_\\wedge$')
+    ax.set_ylim(0.01, 1)
+    ax.plot(chi, coef1.norm_beta_cross())
+    ax.plot(chi, coef2.norm_beta_cross())
+    
+    ax = axarr[1][1]
+    ax.set_title('$\\kappa_\\perp$')
+    ax.set_ylim(0.001, 100)
+    ax.plot(chi, coef1.norm_kappa_perp())
+    ax.plot(chi, coef2.norm_kappa_perp())
+    
+    ax = axarr[2][1]
+    ax.set_title('$\\kappa_\\wedge$')
+    ax.set_ylim(0.01, 10)
+    ax.plot(chi, coef1.norm_kappa_cross())
+    ax.plot(chi, coef2.norm_kappa_cross())
+    
```

Comment:
Ah. Looks like a notebook in the `__main__`. :)

Reply:
Yep, eventually I'll move it to one!